Plugin/Quote/Model/BillingAddressManagement.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Yotpo\SmsBump\Plugin\Quote\Model;
+
+use Magento\Quote\Model\BillingAddressManagement as QuoteBillingAddressManagement;
+use Magento\Quote\Api\Data\AddressInterface;
+
+/**
+ * Class BillingAddressManagement
+ * Plugin to trigger checkout sync
+ */
+class BillingAddressManagement extends AbstractCheckoutTrigger
+{
+    /**
+     * @method afterAssign
+     * @param QuoteBillingAddressManagement $billingAddressManagement
+     * @param int $result
+     * @param int $cartId
+     * @param AddressInterface $address
+     * @param boolean $useForShipping
+     * @return int
+     */
+    public function afterAssign(QuoteBillingAddressManagement $billingAddressManagement, $result, $cartId, AddressInterface $address, $useForShipping = false)
```

Comment:
`$billingAddressManagement` , `$address` , `$useForShipping` are not used here, maybe they can removed

Reply:
Discussed offline.